mappings/net/minecraft/client/render/entity/EntityRenderer.mapping

```diff
@@ -39,7 +39,7 @@ CLASS net/minecraft/class_897 net/minecraft/client/render/entity/EntityRenderer
 		ARG 6 z
 		ARG 8 text
 	METHOD method_3931 getTexture (Lnet/minecraft/class_1297;)Lnet/minecraft/class_2960;
-		ARG 1 boat
+		ARG 1 bee
```

Comment:
seriously?

Reply:
Hey, I didn't do that! Enigma did that for some reason.

Reply:
:trollface:

Reply:
Why was it called `boat` in the first place anyway? Shouldn't it be `entity`?

Reply:
Fixed!